### PR TITLE
pullRequestAdvice Page (Dev-03) TrevorDS

### PR DESCRIPTION
**Pull request by TrevorDS**
<h1>Pull Request Advice Page</h1>

My Branch is "pullRequestAdvice"!

Things I added:
<ul>
<li>docs/course-content/pullRequestAdvice.html</li>
<ul><li>This webpage includes information I deemed valuable to share for anyone wanting to create a Pull Requests on Github.</li></ul>
<li>docs/lib</li>
<ul><li>This would most likely have been created eventually, but I needed to include an image with my webpage, so that is where it ended up at.</li></ul>
</ul>
<br>

Hope it is helpful 😄 🤠 

Here is a live 🔗 to the webpage on my forked Repo: 
<a href="https://trevords.github.io/dev-challenge-01/course-content/pullRequestAdvice.html">Live Webpage</a>

Here is a 🔗 to a nu checker of the Webpage as well (also towards my forked Repo) - all code matches PR code:
<a href="https://validator.w3.org/nu/?doc=https%3A%2F%2Ftrevords.github.io%2Fdev-challenge-01%2Fcourse-content%2FpullRequestAdvice.html"> NU Checker Active Link</a>
<br>

<i>P.S. I forced myself to make the entire webpage via Git CLI & VIM! 😈 </i>

### CODE REVIEW

File: docs/course-content/pullRequestAdvice.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
```
Comment (barrycumbie) [inline docs/course-content/pullRequestAdvice.html:3]: we use Google Style Guide 2 space indents here, good sir!

**Comment by barrycumbie** [comment]
Thx for all the info in this PR...some requests:

1. your branch should be named ```iss99``` where 99 is the actual number. Like this PR is #84. (don't have to edit this one but adhere to in future)
2. Link your issue to this PR over yonder 👉🏼 
3. see my one code comment
4. thx for test link and Nu. Wonder if we can figure out a way to publish an active test site AND nuke your fork into oblivion. (Let's put that on the back burner)
5. ```docs/lib``` yes, neeeeeeded; however, what is ```lib/``` gonna end up lookin' like? Maybe 
```shell
+-- lib/
|     +-- img/
|     +-- scripts/
|     +-- src/
|      |     +--bs4/      
|    +--styles/
etc.
```
6. let's change that filename too, shall we: pull-request-advice.html is better (this is just one step of the PR so...can it be named more precisely?)

File: docs/course-content/pullRequestAdvice.html (hunk 1 above)
Comment (TrevorDS) [reply docs/course-content/pullRequestAdvice.html]: I fixed the spacing!

**Comment by TrevorDS** [comment]
<h1>Response</h1>
<ol>
<li>✔️ Will keep this in mind!</li>
<li>✔️ Issue #89</li>
<li>✔️ Fixed my code spacing / new img src based on new lib structure.</li>
<li>✔️ Will remove fork (later on)!</li>
<li>✔️ Fixed docs/lib/ dir structure.</li>
<li>✔️ Corrected file name.</li>
</ol>

🤠 😄 😈

**Comment by barrycumbie** [comment]
image

**Pull request merged**